server/service/src/processors/transfer/requisition/mod.rs

```diff
@@ -75,13 +75,22 @@ pub(crate) fn process_requisition_transfers(
     let active_stores =
         ActiveStoresOnSite::get(&ctx.connection).map_err(Error::GetActiveStoresOnSiteError)?;
 
+    let name_ids = active_stores.name_ids();
+    let name_link_repo = NameLinkRowRepository::new(&ctx.connection);
+    let name_link_ids: Vec<String> = name_link_repo
+        .find_many_by_name_ids(name_ids.as_slice())
+        .unwrap() // MERGE: handle error
```

Comment:
I'd like advice on how I should handle this!

Reply:
What do you mean? the database error? Do:

```rust
.find_many_by_name_ids(name_ids.as_slice())
.map_err(Error::DatabaseError)?
```